core/common/src/main/java/alluxio/Seekable.java

```diff
@@ -26,4 +26,9 @@
    * @param pos the position to seek to, it must be between 0 and the end of the stream - 1
    */
   void seek(long pos) throws IOException;
+
+  /**
+   * @return the current position of the stream
+   */
+  long getPos();
```

Comment:
there might be cases for UFS files that you can `getPos` but you can not `seek`, right? e.g., files on tape devices...